### ISSUE TITLE
Update T&C copy

### ISSUE THREAD

**Issue by rachelhamlin** [status-im/ens-usernames#40]
Sonja reviewed the new T&C copy and had no legal feedback. 

Please update the current T&C copy to match the text & formatting in this file:

https://hackmd.io/CIX8p47nR5ikuQQqbQ7NDg?view

**Comment by obi2020** [comment]
I made some edits here https://hackmd.io/CIX8p47nR5ikuQQqbQ7NDg?both

Of note: in the privacy policy, I changed to address user directly to make it more readable/actionable

There are outstanding questions:

- Uppercase letters (can't the dapp just disallow them from registering in the first place?)
- "Users are free to own the subdomains, but they are property of Status Network" I edit to `free to use` but maybe I'm not comprehending properly what this means. 
- What is meant by fingerprint?
- "Your contact code is associated… which is publicly known" is not clear. As is, seems to contradict what is stated above, that user has control over a "secret" - need to make clearer what's public and what user can control.
